geodiff/CMakeLists.txt

```diff
@@ -7,11 +7,15 @@ SET(CMAKE_CXX_VISIBILITY_PRESET hidden)
 SET(CMAKE_VISIBILITY_INLINES_HIDDEN 1)
 SET(CMAKE_CXX_STANDARD 11)
 
+# set path to additional CMake modules
+SET(CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake ${CMAKE_MODULE_PATH})
+
 SET(ENABLE_TESTS TRUE CACHE BOOL "Build tests?")
 SET(ENABLE_COVERAGE FALSE CACHE BOOL "Enable GCOV code coverage?")
 SET(BUILD_TOOLS TRUE CACHE BOOL "Build tool executables?")
 SET(BUILD_STATIC_LIBS FALSE CACHE BOOL "Build only static libraries?")
 SET(WITH_INTERNAL_SQLITE3 TRUE CACHE BOOL "Use internal copy of SQLITE3")
```

Comment:
I would remove internal sqlite3 option at all and rename it to "with_sqlite3" similar to postgres

Reply:
yes, but that can be only done when we get rid of the hard dependency on sqlite3 session ext...